metrics/web/lib/project_groups/presentation/state/project_groups_notifier.dart

```diff
@@ -187,21 +191,28 @@ class ProjectGroupsNotifier extends ChangeNotifier {
     notifyListeners();
   }
 
+  /// Sets the [ProjectGroupDialogViewModel] to null.
+  void resetProjectGroupDialogViewModel() {
+    _projectGroupDialogViewModel = null;
+  }
+
+  /// Sets the project name filter to null.
   void resetFilterName() {
     _projectNameFilter = null;
   }
 
-  /// Changes checked status to [isChecked] for the [ProjectCheckboxViewModel]
-  /// by [projectId].
-  void toggleProjectCheckedStatus({String projectId, bool isChecked}) {
-    if (projectId == null && isChecked == null) return;
+  /// Changes a checked status to an opposite for the [ProjectCheckboxViewModel]
+  /// by the [projectId].
+  void toggleProjectCheckedStatus({String projectId}) {
```

Comment:
Nit: should we make it a positional argument?
```suggestion
  void toggleProjectCheckedStatus(String projectId) {
```
or add required?
```suggestion
  void toggleProjectCheckedStatus({@required String projectId}) {
```